ui/Screen/Org/Projects.svelte

```diff
@@ -41,20 +57,65 @@
     max-width: var(--content-max-width);
     min-width: var(--content-min-width);
   }
+
+  .pending {
+    margin-bottom: 2rem;
+  }
+
+  .header {
+    display: flex;
+    padding: 0.5rem 3rem 0.5rem;
+    width: 100%;
+  }
 </style>
 
 <div class="container">
-  {#if anchoredProjects.length !== 0 || unresolvedAnchors.length !== 0}
-    <ProjectList
-      projects={anchoredProjects}
-      userUrn={session.identity.urn}
-      on:select={select} />
-
-    <UnresolvedAnchorList anchors={unresolvedAnchors} />
+  {#if pendingResolved.length !== 0 || pendingUnresolved.length !== 0}
+    <div class="pending">
+      <div class="header">
+        <p class="typo-text-bold">Pending</p>
+        <p style="margin-left: .5rem; color: var(--color-foreground-level-6);">
+          Not enough of the members have signed this anchor.
+        </p>
+      </div>
+      {#if pendingResolved.length !== 0}
+        <ProjectList
+          projects={pendingResolved}
+          userUrn={session.identity.urn}
+          on:select={select} />
+      {:else if pendingUnresolved.length !== 0}
+        <UnresolvedAnchorList anchors={pendingUnresolved} />
+      {/if}
+    </div>
+  {/if}
+  {#if confirmedResolved.length !== 0 || confirmedUnresolved.length !== 0}
+    {#if pendingResolved.length !== 0 || pendingUnresolved.length !== 0}
+      <div class="header">
+        <p class="typo-text-bold">Confirmed</p>
+        <p style="margin-left: .5rem; color: var(--color-foreground-level-6);">
+          These projects have been anchored and signed by the org.
+        </p>
+      </div>
+    {/if}
+    {#if confirmedResolved.length !== 0}
+      <ProjectList
+        projects={confirmedResolved}
+        userUrn={session.identity.urn}
+        on:select={select} />
+    {/if}
+    {#if confirmedUnresolved.length !== 0}
+      <div class="header">
+        <p style="color: var(--color-foreground-level-6);">
+          These anchored projects haven't been found in your network yet, try
+          following them.
+        </p>
+      </div>
+      <UnresolvedAnchorList anchors={confirmedUnresolved} />
+    {/if}
   {:else}
     <EmptyState
```

Comment:
This should only be shown if there are no anchors at all. Fixed in 5518ed210d262ff6090d31f4369114c76e471e3a.

<img width="1552" alt="Screenshot 2021-07-01 at 09 45 50" src="https://user-images.githubusercontent.com/158411/124093771-716edc00-da58-11eb-9e5c-3d1b9fce98e7.png">